vector/src/Data/Vector/Strict.hs

```diff
@@ -23,11 +23,9 @@
 -- Immutable strict boxed vectors (that is, polymorphic arrays capable
 -- of holding any Haskell value). It is possible to create vector
 -- which contain bottom elements, either by using mutable interfaces
```

Comment:
Can drop "either" here now, also might be good to clarify the vectors that might contain bottom elements are the original data type, not this strict type.

Reply:
I reworded module comment in order to make it more clear